docs-src/typescript/chapter-project-setup/choose-dev-cluster.md

```diff
@@ -0,0 +1,133 @@
+
+However, the local development server does not emit any metrics.
+If you are eager to to set up Cluster-level metrics for performance tuning, we recommend using a self-hosted Cluster or Temporal Cloud.
+
+#### Start the dev server
+
+**How to start a local development server**
+
+If you have successfully installed the Temporal CLI, open a new terminal and run the following command:
+
+```bash
+temporal server start-dev
+```
+
+This command automatically starts the Temporal Web UI, creates a default Namespace, and creates an in-memory database.
+
+The Temporal Web UI serves to [http://localhost:8233](http://localhost:8233/).
+
+For more command details and options, see the [CLI reference](/cli/server/start-dev)
+
+#### Create a custom Namespace
+
+**How to create a Namespace on the development server**
+
+The development server does automatically create a default Namespace (named "default") when it starts up.
+However, you will create a custom one for our application.
+Since this is something recommended at a production level, it's recommend practicing it with the development server.
+
+Use the `temporal operator namespace create` command using the Temporal CLI to create a Namespace on the development server.
+
+```bash
+temporal operator namespace create backgroundcheck_namespace
+```
+
+For command details and options, see the [CLI reference](/cli/operator/namespace/create).
+
+### Temporal Cloud
+
+**When to use Temporal Cloud**
+
+If you do not have a Temporal Cloud Account, you can request one using the link on the [Get started with Temporal Cloud](https://docs.temporal.io/cloud/get-started) guide.
+
+We recommend starting off with Temporal Cloud if you already have a production use case, or need to move a scalable proof of concept into production.
+
+In other words, Temporal Cloud is perfect if you are ready to run at scale and don’t want the overhead of managing your own self-hosted Cluster.
+
+To create a Namespace in Temporal Cloud, follow the instructions in [How to create a Namespace](/cloud/namespaces-create).
+
+:::info Safely store your certificate and private key
+
+Store certificates and private keys generated for your Namespace as files or environment variables in your project.
+You need access to your certificate and key to run your Workers and start Workflows.
+
+For more information on certificate requirements, see [How to manage certificates in Temporal Cloud](/cloud/certificates-intro).
+
+:::
+
+### Self-hosted Temporal Cluster
+
+We recommend using a self-hosted environment if you are starting something new and need to scale with production-level features, but don’t yet need or want to pay for Temporal Cloud.
+
+For example, running a self-hosted Cluster lets you try different databases, view Cluster metrics, use custom [Search Attributes](/concepts/what-is-a-search-attribute), and even play with the [Archival](/concepts/what-is-archival) feature.
+
+It's completely possible to run a production level self-hosted Cluster.
```

Comment:
This is in a couple different dev guides right now and the use of "completely" here feels a bit off -- it's one of those adverbs that accidentally sounds more caveated by its inclusion.

Reply:
(Also in Python)

Reply:
Ya, we can nix the completely.